desc/grid.py

```diff
@@ -585,12 +611,7 @@ def _create_nodes(self, nodes):
         # This may cause the surface_integrals() function to fail recognizing
         # surfaces outside the interval [0, 2pi] as duplicates. However, most
         # surface integral computations are done with LinearGrid anyway.
-        spacing = (  # make weights sum to 4pi^2
```

Comment:
The three lines above about surface integrals can be deleted now. (But keep the note to not modulo custom nodes please).